backend/src/main/java/com/htecgroup/skynest/model/request/UserRegisterRequest.java

```diff
@@ -1,14 +1,37 @@
 package com.htecgroup.skynest.model.request;
 
 import lombok.Data;
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Pattern;
+import javax.validation.constraints.Size;
 
 @Data
 public class UserRegisterRequest {
 
+  @Email(message = "format is not valid", regexp = "(.+)@(\\S+)\\.(\\S+)")
+  @Size(max = 254, message = "length cannot be over 254 characters")
   private String email;
+
+  @Pattern(
+      regexp = "(?=.*[a-z])(?=.*[A-Z])(?=.*\\d)[a-zA-Z\\d]{8,50}",
+      message = "format not valid")
   private String password;
+
+  @NotBlank(message = "cannot be empty")
```

Comment:
Can it be Null ? Applies to all required properties

Reply:
@NotBlank forbid null values. So I can add that in message. For example Cannot be null or empty.